test/parallel/test-whatwg-url-properties.js

```diff
@@ -142,3 +142,19 @@ assert.strictEqual(url.searchParams, oldParams);
   assert.strictEqual(opts.search, '?l=24');
   assert.strictEqual(opts.hash, '#test');
 }
+
+// Test special origins
+[
+  { expected: 'https://whatwg.org',
```

Comment:
I think most tests here are already in `url-tests.js`, therefore tested by [`test-whatwg-url-origin`](https://github.com/nodejs/node/blob/master/test/parallel/test-whatwg-url-origin.js)?